contracts/secret-storage/SecretManager.sol

```diff
@@ -518,13 +548,7 @@ contract SecretManager is
         _updateUsdcDepositPostPayment(_secretId, _enclaveAddress, enclaveIndex);
 
         uint256 sizeLimit = userStorage[_secretId].sizeLimit;
-        if(block.timestamp > userStorage[_secretId].endTimestamp) {
-            _removeSelectedEnclave(_secretId, enclaveIndex);
-            if(userStorage[_secretId].selectedEnclaves.length == 0)
-                _refundExcessDepositAndRemoveStore(_secretId);
-        }
-        else
-            _replaceStore(_secretId, _enclaveAddress, enclaveIndex, true);
+        _replaceStore(_secretId, _enclaveAddress, enclaveIndex, true);
```

Comment:
what's the reason to remove this condition?

Reply:
`_replaceStore()` already had the code for if case, so removed the duplicate code from here.